python/lsst/ts/ofc/SubSysAdap.py

```diff
@@ -73,10 +69,9 @@ def _getRotMatMirror(self, configDir, dofGroup, rotMatfileName):
         rotMatFilePath = os.path.join(configDir, mirrorDirName, rotMatfileName)
         rotMatFile = ParamReader(filePath=rotMatFilePath)
 
-        rotMatBend = rotMatFile.getSetting("bend")
         rotMatAct = rotMatFile.getSetting("act")
 
-        return np.diag(rotMatBend), rotMatAct
+        return rotMatAct
```

Comment:
Rather change to "return rotMatFile.getSetting…."

Reply:
Fixed in the commit: a2945e0. Thanks!